modules/codegen/src/main/java/org/apache/ignite/codegen/SystemViewRowAttributeWalkerGenerator.java

```diff
@@ -148,7 +148,8 @@ public static void main(String[] args) throws Exception {
      */
     private <T> Collection<String> generate(Class<T> clazz) {
         final List<String> code = new ArrayList<>();
-        final Set<String> imports = new TreeSet<>();
+        Comparator<String> importComparatror = (s1, s2) -> s1.replace(";","").compareTo(s2.replace(";",""));
```

Comment:
@aleksey-kurinov There is typo in the var name. There is no white space between parameters. (See [CodingGuidelines](https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines))

I suggest to pass comparator to the constructor parameter:
`new TreeSet<>(Comparator.comparing(s -> s.replace(";", "")))`